third_party/2and3/itsdangerous.pyi

```diff
@@ -1,16 +1,8 @@
+import sys
 from datetime import datetime
 from typing import Any, Callable, IO, MutableMapping, Optional, Text, Tuple, TypeVar, Union
 
-PY2: bool
-text_type = str
-int_to_byte = Callable[[int], bytes]
-number_types = (int, float)
-izip = zip
-
-_bytes_like = Union[bytearray, bytes]
-_str_like = Union[str, bytes]
-_can_become_bytes = Union[str, bytes, bytearray]
-_comparable_bytes = TypeVar('_comparable_bytes', str, _bytes_like)
+_C = TypeVar('_C', str, bytes)
```

Comment:
Should this just be AnyStr? The current definition reduces to `TypeVar('_C', str, str)` in Python 2, which doesn't seem like it should work.

Reply:
Well, I removed it in the following commit, since it's not needed anymore, so I guess this is moot.